source/localizable/tutorial/acceptance-test.es-ES.md

```diff
@@ -69,7 +69,7 @@ Now run your test suite with the CLI command, `ember test --server`.
 
 By default, when you run `ember test --server`, Ember CLI runs the [Testem test runner](https://github.com/testem/testem), which runs Qunit in Chrome and [PhantomJS](http://phantomjs.org/).
 
-Our launched Chrome web browser now shows 10 successful tests. If you toggle the box labeled "Hide passed tests", you should see our successful acceptance test, along with 9 passing JSHint tests. Ember tests each file you create for syntax issues (known as "linting") using [JSHint](http://jshint.com/).
+Our launched Chrome web browser now shows 10 successful tests. If you toggle the box labeled "Hide passed tests", you should see our successful acceptance test, along with 9 passing ESLint tests. Ember tests each file you create for syntax issues (known as "linting") using [ESLint](http://eslint.org/).
```

Comment:
Would you remind removing these changes?  These localized files get updated automatically for us ...